README.md

```diff
@@ -1,29 +1,32 @@
 # CoAP CLI
-Simple CoAP cli client written in Go.
 
+Simple CoAP cli client written in Go.
 
 ## Usage
+
 Pre-built binary can be found here: https://github.com/mainflux/coap-cli/releases/tag/v0.3.3.
 When running, please provide following format:
 `coap-cli` followed by method code (`get`, `put`, `post`, `delete`) and CoAP URL. After that, you can pass following flags:
 
-| Flag | Description                                    | Default value    |
-| ---- | ---------------------------------------------- | ---------------- |
-| o    | observe   option - only valid with GET request | false            |
-| auth | auth option sent as URI Query                  | ""               |
-| h    | host                                           | "localhost"      |
-| p    | port                                           | "5683"           |
-| d    | data to be sent in POST or PUT                 | ""               |
-| cf   | content format                                 | 50 (JSON format) |
+| Flag | Description                                  | Default value    |
```

Comment:
Paste the help message from the cli for example https://docs.magistrala.abstractmachines.fr/cli/

Reply:
This is what I meant https://github.com/absmach/magistrala-docs/blob/c8a7b8c04a3d57eecde616f475bdb0ff739e26b1/docs/cli.md?plain=1#L23C1-L70C4

Reply:
![image](https://github.com/absmach/coap-cli/assets/28790446/aa4037c7-6b6b-4d4d-b9b7-15a4cfc0806f)

No need for this